automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -338,7 +338,7 @@ trait Orchestration extends FireCloudClient with LazyLogging with SprayJsonSuppo
       val trialProjects: TrialProjects = countTrialProjects()
       if (trialProjects.available < count) {
         postRequest(apiUrl(s"api/trial/manager/projects?operation=create&count=${count - trialProjects.available}"))
-        retry(30.seconds, 10.minutes)({
+        retry(30.seconds, 20.minutes)({
```

Comment:
This file is moving into workbench-libs. I think this function probably doesn't need to go with it. Would it make sense to be in the test spec that needs it?